src/DataWrangling/EN4/EN4_metadata.jl

```diff
@@ -38,43 +44,32 @@ end
 
 default_download_directory(::EN4Monthly) = download_EN4_cache
 
-datasetstr(md::EN4Metadata) = string(md.dataset)
-
-datestr(md::EN4Metadata) = string(first(md.dates), "--", last(md.dates))
-datestr(md::EN4Metadatum) = string(md.dates)
-
-Base.summary(md::EN4Metadata) = string("EN4Metadata{", datasetstr(md), "} of ",
-                                        md.name, " for ", datestr(md))
-
+metaprefix(::EN4Monthly) = "EN4Metadata"
 Base.size(data::Metadata{<:EN4Monthly}) = (360, 173, 42, length(data.dates))
-
 Base.size(::Metadatum{<:EN4Monthly}) = (360, 173, 42, 1)
 
 # The whole range of dates in the different dataset datasets
 all_dates(::EN4Monthly, name) = DateTime(1900, 1, 1) : Month(1) : DateTime(2024, 12, 1)
 
-# File name generation specific to each Dataset dataset
+# Note, EN4 files contain all variables, so the filenames do not
+# depend on metadata.name.
 function metadata_filename(metadata::Metadatum{<:EN4Monthly})
-    shortname = short_name(metadata)
     yearstr  = string(Dates.year(metadata.dates))
     monthstr = string(Dates.month(metadata.dates), pad=2)
     return "EN.4.2.2.f.analysis.g10." * yearstr * lpad(string(monthstr), 2, '0') * ".nc"
 end
 
 # Convenience functions
-short_name(data::Metadata{<:EN4Monthly}) = EN4_short_names[data.name]
-
+dataset_variable_name(data::EN4Metadata) = EN4_dataset_variable_names[data.name]
 location(data::EN4Metadata) = EN4_location[data.name]
+dataset_temperature_units(data::EN4Metadata) = Kelvin()
+absolute_latitude_bounds(data::Metadata{<:EN4Monthly, <:Any}) = (-83.5, 89.5)
```

Comment:
A thought: I think what we really want is a way to reproduce the native grid of each dataset... that's where `absolute_latitude_bounds ` is used for, right? perhaps we implement a `native_grid(dataset)` method and call that within, e.g., 

https://github.com/CliMA/ClimaOcean.jl/blob/6ad11ed62e8f80baa3a67b8b4279b401ac2bb1ce/src/DataWrangling/metadata.jl#L233

Reply:
yes, I think that would be better! we actually need to go in that direction, because for GLORYS we do not always want to assume that the longitude bounds are 0, 360 (for example), we need to do limited area domains.

Reply:
Perhaps a later PR. Either or.